### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by RobertHebel**

### CODE REVIEW

File: frontend/src/components/common/comboBox.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@import '../../constants/colors';
+
+$error-color: $PERSIAN_PINK;
```
Comment (adam-golab) [inline frontend/src/components/common/comboBox.scss]: Why didn't you defined those variables in `colors.scss`? Now there is much redundancy. Every file begins with `$error-color: $PERSIAN_PINK;` and so on.

File: frontend/src/components/sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+$sidebar-item-color: $TAPA;
+$sidebar-item-background-hover: $SILVER;
+
+.sidebar {
+  background-color: $sidebar-background-color;
+  display: flex;
+  flex-flow: column wrap;
+  flex-wrap: wrap;
+  align-content: stretch;
+  min-height: calc(100vh - 8.45em);
```
Comment (adam-golab) [inline frontend/src/components/sidebar.scss]: Is this `calc` necessary?

File: frontend/src/components/common/comboBox.scss (hunk 1 above)
Comment (RobertHebel) [inline frontend/src/components/common/comboBox.scss]: https://github.com/brainhubeu/brainfeed/pull/17#discussion_r126922249

File: frontend/src/components/sidebar.scss (hunk 2 above)
Comment (RobertHebel) [inline frontend/src/components/sidebar.scss]: I think that's the easiest way to ensure that sidebar have a 100% height of the browser

**Pull request merged**